src/Silakov.Homework5/Managers/ZooManager.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Silakov.Homework5.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Silakov.Homework5.Managers
+{
+    class ZooManager
+    {
+        /// <summary>
+        /// Список животных зоопарка.
+        /// </summary>
+        private List<AnimalBase> Animals { get; set; }
```

Comment:
looks more like a field, because there is no additional logic getting and setting it and the purpose is just encapsulation of an animals list